asp_interactive_configuration/ooasp/config.py

```diff
@@ -254,17 +273,34 @@ def attribute_values(self)->List[Predicate]:
         """
         return list(self.fb.query(self.UNIFIERS.AttributeValue).all())
 
+    @property
+    def int_attributes(self)->List[Predicate]:
+        """
+        The list of attribute values
+        """
+        return list(self.fb.query(self.UNIFIERS.AttributeInt).all())
+
+
     @property
     def user_input(self)->List[Symbol]:
         """
         The of symbols inside a user predicate
         """
         input_facts = list(self.fb.query(self.UNIFIERS.User).select(self.UNIFIERS.User.predicate).all())
-        return [f.symbol for f in input_facts]
+        user_fb  = unify(self.editable_unifiers, [f.symbol if type(f)==Raw else f for f in input_facts])
+        
+        return user_fb
 
-    def associated_by(self, obj, assoc_name)->List[Predicate]:
+    def associated_by(self, obj:int, assoc_name:str)->List[Predicate]:
         """
         The list of associations from object obj via association assoc_name
+
+        Parameters:
+            obj (int): Object where the association starts, can be right ot left
```

Comment:
Is it correct that the integers 1 and 2 refer to the left and right end, respectively? If yes, please add this information here.

Reply:
No, it is the id of the. The object can be either left or right of the association. I just rephrased it to be more clear